### PR TITLE
License + codecov

### PR DESCRIPTION
**Pull request by jgrund**
This patch updates the readme file + adds a license file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 language: python
 python:
-    - "2.7"
+  - "2.7"
+before_script:
+  - pip install codecov
```
Comment (tanabarr) [inline .travis.yml]: I assume there's a reason why this can't be added in the requirements.tx

File: README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-license that can be found in the LICENSE file.
+# iml-common
+
+[![Build Status](https://travis-ci.org/intel-hpdd/iml-common.svg?branch=master)](https://travis-ci.org/intel-hpdd/iml-common)
+[![codecov](https://codecov.io/gh/intel-hpdd/iml-common/branch/master/graph/badge.svg)](https://codecov.io/gh/intel-hpdd/iml-common)
 
 iml-common is a python library package required by the agent and manager.
 
-[![Build Status](https://travis-ci.org/intel-hpdd/iml-common.svg?branch=master)](https://travis-ci.org/intel-hpdd/iml-common)
+
```
Comment (tanabarr) [inline README.md]: do we need this blank line and the one before?

File: .travis.yml (hunk 1 above)
Comment (jgrund) [inline .travis.yml]: It doesn't make sense there. `requirements.txt` is also installed locally. The codecov package is only relevant in CI.

**Comment by jgrund** [comment]
Landed in: https://github.com/intel-hpdd/iml-common/commit/7b6b8185f2625ac6717d76d762a1121093e95213

**Pull request closed without merging**